low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -265,8 +231,10 @@ def prepare_mem_json(self, status, description):
                         "slab": total_memory['slab']
                     }
                 ]
-            }
-        }
+        memory_dict = self.get_health_template(uid, is_fru=False)
+        self.set_health_data(
+            memory_dict, status=status, description=description,
+            specifics=specifics)
         return memory_dict
 
 
```

Comment:
Since we added unittest, please remove below block of code under `if __name__ == "__main__"`
Same for storage.py